### DISCUSSION TITLE
Modify max column and row in product blocks

### DISCUSSION THREAD

**Discussion by teshn** [woocommerce/woocommerce#45019] [WooCommerce, Blocks, and Site Editing]
Hey folks,

I still search for a somewhat easy way to increase the maximum products shown for e.g. "Products by Category". Either by increasing the row/columns or pagination regarding performance purposes. 

ref: https://wordpress.org/support/topic/only-the-outdated-version-of-a-plugin-works/#post-11826679

Solution provided in the last answer does not work for me

_functions.php_
`function storefront_child_add_woocommerce_support() {
    add_theme_support( 'woocommerce' );
}
add_action( 'after_setup_theme', 'storefront_child_add_woocommerce_support' );

add_theme_support('woocommerce',['product_blocks' => ['max_rows' => 10]]);`

Is it meanwhile possible to alter the max row/column for e.g. overwriting a file / template? Or do I need to look up how to create custom Gutenberg Blocks?

**Comment by nerrad** [comment]
Hi there 👋🏻 

We're currently working on a new block (Product Query) that will be a variation of the WordPress core Query Loop block and overtime will be using that to power all the existing product blocks. This will bring a number of changes including potentially allowing for increasing the number of rows/columns (as well as introducing pagination).

**Reply by teshn** [reply]
Thank you for the reply. When can we except the new features? Couple of months?
